Procurement/ViewModel/Recipes/SameBaseTypeRecipe.cs

```diff
@@ -34,30 +34,47 @@ public override string Name
 
         public override string GetResultName(RecipeResult result)
```

Comment:
We should have a constructor parameter that takes a "Long name" and a "Short name" for every recipe otherwise people will forget to provide on when a recipe is implemented. This should be applied to all existing recipes.

Reply:
Honestly, I'm only making this PR so I don't have to keep rebasing a local change to shorten these recipe names for myself.  Also, especially since each recipe can have arbitrarily many names/descriptions, we would need to take lists of short and long names in the constructor (or something more complicated to ensure the long and short names match up to each other).  Moreover, the short names are obfuscated, so I would rather not encourage using them unless we have to (that is, unless they're too long).

Reply:
That's a fair comment for sure, I ran I to the long name issue with the shaper elder set recipe's. Similar problem. Maybe reworking the UI somehow may be a longer term solution.

Reply:
Ah, right, the elder/shaper sets have long names.  I'll add short names for them.

I agree that a UI rework is probably a better longer term solution.  I just don't want to figure out how to do any of that right now. :persevere:

Reply:
Alright, I added short names for each of the `RareSetRecipe` instances.